edx_proctoring/admin.py

```diff
@@ -334,6 +342,183 @@ def save_model(self, request, review, form, change):
+        else:
+            return queryset
+
+
+class ProctoredExamAttemptForm(forms.ModelForm):
+    """
+    Admin Form to display for reading/updating a Proctored Exam Attempt
+    """
+
+    class Meta(object):  # pylint: disable=missing-docstring
+        model = ProctoredExamStudentAttempt
+        fields = '__all__'
+
+    STATUS_CHOICES = [
+        (ProctoredExamStudentAttemptStatus.eligible, _('Eligible')),
+        (ProctoredExamStudentAttemptStatus.created, _('Create')),
+        (ProctoredExamStudentAttemptStatus.download_software_clicked, _('Download Software Clicked')),
+        (ProctoredExamStudentAttemptStatus.ready_to_start, _('Ready To Start')),
+        (ProctoredExamStudentAttemptStatus.started, _('Started')),
+        (ProctoredExamStudentAttemptStatus.ready_to_submit, _('Ready To Submit')),
+        (ProctoredExamStudentAttemptStatus.declined, _('Declined')),
+        (ProctoredExamStudentAttemptStatus.timed_out, _('Time Out')),
+        (ProctoredExamStudentAttemptStatus.submitted, _('Submitted')),
+        (ProctoredExamStudentAttemptStatus.second_review_required, _('Second Review Required')),
+        (ProctoredExamStudentAttemptStatus.verified, _('Verified')),
+        (ProctoredExamStudentAttemptStatus.rejected, _('Rejected')),
+        (ProctoredExamStudentAttemptStatus.not_reviewed, _('Not Reviewed')),
+        (ProctoredExamStudentAttemptStatus.error, _('Error')),
+    ]
+
+    status = forms.ChoiceField(choices=STATUS_CHOICES)
+
+
+class ProctoredExamStudentAttemptAdmin(admin.ModelAdmin):
+    """
+    Admin panel for Proctored Exam Attempts
+    """
+
+    readonly_fields = [
+        'user',
+        'proctored_exam',
+        'started_at',
+        'completed_at',
+        'last_poll_timestamp',
+        'last_poll_ipaddr',
+        'attempt_code',
+        'external_id',
+        'allowed_time_limit_mins',
+        'taking_as_proctored',
+        'is_sample_attempt',
+        'student_name',
+        'review_policy_id',
+        'is_status_acknowledged'
+    ]
+
+    list_display = [
+        'username',
+        'exam_name',
+        'course_id',
+        'attempt_code',
+        'status',
+        'modified_timestamp'
+    ]
+
+    search_fields = [
+        'user__username',
+        'attempt_code'
+    ]
+
+    list_filter = [
+        ExamAttemptFilterByStatus,
+        "taking_as_proctored",
+        "is_sample_attempt",
+        ExamAttemptFilterByCourseId
+    ]
+
+    form = ProctoredExamAttemptForm
+
+    def username(self, obj):
+        """ Return user's username of attempt"""
+        return obj.user.username
+
+    def exam_name(self, obj):
+        """ Return exam_name of attempt"""
+        return obj.proctored_exam.exam_name
+
+    def course_id(self, obj):
+        """ Return course_id of attempt"""
+        return obj.proctored_exam.course_id
+
+    def modified_timestamp(self, obj):
+        """ Return modified timestamp of attempt"""
+        return time.mktime(obj.modified.timetuple())
```

Comment:
I guess the linux timestamp would not be useful here. You should probably use the obj.modified by itself to show a human readable date/time.